.github/workflows/swift-toolchain.yml

```diff
@@ -3193,6 +3193,34 @@ jobs:
             Install-Package -Name WixToolset.Sdk -RequiredVersion 4.0.1 -Force
           }
 
+      - uses: actions/checkout@v4
+        if: matrix.arch == 'amd64'
+        with:
+          repository: microsoft/mimalloc
+          ref: refs/tags/v2.1.7
```

Comment:
Can we pull this out into a global like we have done for the other dependencies (thanks to @Steelskin!).

Reply:
Done